uefi/src/boot.rs

```diff
@@ -162,6 +164,50 @@ pub unsafe fn free_pool(ptr: NonNull<u8>) -> Result {
     unsafe { (bt.free_pool)(ptr.as_ptr()) }.to_result()
 }
 
+/// Creates an event
```

Comment:
For the sake of progress, you can also do this in a follow-up.

```suggestion
/// Creates an event.
```